tools/generate-helpers/pg-table-bindings/store_test.go.tpl

```diff
@@ -160,10 +160,11 @@ func (s *{{$namePrefix}}StoreSuite) TestStore() {
 const (
 	withAllAccess = "AllAccess"
```

Comment:
I don't think this is formatted correctly, but I guess there is no convenient way to run `go fmt` on the template...